extension/src/controllers/subtitle-controller.ts

```diff
@@ -175,21 +208,8 @@ export default class SubtitleController {
         return settings.subtitleBlur ? 'asbplayer-subtitles-blurred' : '';
     }
 
-    set subtitleAlignment(value: SubtitleAlignment) {
-        if (this._subtitleAlignment === value) {
-            return;
-        }
-
-        const { subtitleOverlayParams, notificationOverlayParams } = this._elementOverlayParams(value);
-        this._applyElementOverlayParams(this.subtitlesElementOverlay, subtitleOverlayParams);
-        this._applyElementOverlayParams(this.notificationElementOverlay, notificationOverlayParams);
-        this.subtitlesElementOverlay.hide();
-        this.notificationElementOverlay.hide();
-        this._subtitleAlignment = value;
-    }
-
-    get subtitleAlignment() {
-        return this._subtitleAlignment;
+    private _getSubtitleTrackAlignment(trackIndex: number) {
+        return this.subtitleTrackAlignments[trackIndex] || 'bottom';
```

Comment:
As mentioned above the default value should really be the setting for the 0th track:

```
this.subtitleTrackAlignments[trackIndex] || this.subtitleTrackAlignments[0]
```